src/roles/common/tasks/fix-image-paths.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Get unzipped files dir
+  set_fact:
+    nuage_unzipped_files_dir: "{{ common.nuage_unzipped_files_dir }}"
+
+- name: Replace non-relative path
+  set_fact:
+    nuage_unzipped_files_dir: "{{ nuage_unzipped_files_dir | regex_replace('.+?(?=metroae_data)', '/') }}"
+  when: common.nuage_unzipped_files_dir | regex_search("/metroae_data/") and role_path | regex_search("^/source/nuage-metroae")
+
+- name: Replace relative path
+  set_fact: 
+    nuage_unzipped_files_dir: "{{ '/metroae_data/' + nuage_unzipped_files_dir }}"
+  when: common.nuage_unzipped_files_dir | regex_search("^[^/]") and role_path | regex_search("^/source/nuage-metroae")
+
+- name: Check if directory needs to be specified to be under metroae_data installation mount directory
+  assert:
+    that:
+    - common.nuage_unzipped_files_dir | regex_search("/metroae_data/")
+    fail_msg: The directory needs to be specified under the /metroae_data/ installation mount directory from the container installation
+  when: role_path | regex_search("^/source/nuage-metroae")

```

Comment:
Better to move this and the above 2 tasks into a "- block:" and have a single "when:" to check the role_path condition.